grep-0020-restructure_blocks.md

```diff
@@ -67,6 +68,7 @@ diffs for illustration; generated by manually cut'n'pasting text generated by
 * gr-streamops: things like stream_to_vector, head, you get the idea.  zero-deps
 * gr-logical kind of on the border to gr-math, should maybe get the packed/unpacked? zero-deps
 * gr-filter gets the moving_average, which for some reason is in gr-blocks
+* gr-pdu: create new in tree module for blocks that operate on PDUs or convert to/from PDUs
```

Comment:
Perhaps should make the distinction here that gr-pdu is for generic pdu utility blocks, not for pdu versions of signal processing blocks

Reply:
agree. i have also removed the blocks that are 'add this one new block to gr-filter' from this GREP